grapple/types/sites.py

```diff
@@ -82,12 +83,16 @@ def resolve_sites(self, info, **kwargs):
         def resolve_site(self, info, **kwargs):
             id, hostname = kwargs.get("id"), kwargs.get("hostname")
 
-            try:
-                if id:
-                    return Site.objects.get(pk=id)
-                elif hostname:
-                    return Site.objects.get(hostname=hostname)
-            except BaseException:
-                return None
+            if id:
+                return Site.objects.filter(pk=id)
```

Comment:
note: this should be `.get()`, otherwise we get a `Received incompatible instance \"<QuerySet [<Site: grapple.test:8000 [default]>]>` error

Reply:
Oops, I forgot to add `.first()` here. I used this instead of `.get()` so that it returns `None` if the site is not found (replicating what `except BaseException:` was doing before)